src/sidebar/store/modules/selection.js

```diff
@@ -138,33 +95,37 @@ function init(settings) {
     highlighted: {},
 
     filterQuery: settings.query || null,
-    selectedTab: TAB_DEFAULT,
     focusMode: {
       enabled: settings.hasOwnProperty('focus'),
       focused: true,
       // Copy over the focus confg from settings object
       config: { ...(settings.focus ? settings.focus : {}) },
     },
 
+    selectedTab: uiConstants.TAB_DEFAULT,
```

Comment:
`TAB_DEFAULT` is not used anywhere outside of this file so we could just remove that constant and use `uiConstants.TAB_ANNOTATIONS` directly here.